src/main/java/racingcar/domain/car/Cars.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.domain.car;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import racingcar.domain.car.strategy.ForwardStrategy;
+import racingcar.domain.car.validator.CarsValidator;
+import racingcar.dto.CarDto;
+
+public class Cars {
+
+    private final List<Car> cars = new ArrayList<>();
+    private final ForwardStrategy forwardStrategy;
+
+    public Cars(final List<String> carNames, final ForwardStrategy forwardStrategy) {
+        carNames.forEach(carName -> cars.add(new Car(carName)));
+        CarsValidator.validateCarNames(carNames);
+        this.forwardStrategy = forwardStrategy;
+    }
+
+    public void moveForwardOrStop() {
+        cars.forEach(car -> car.moveForward(forwardStrategy));
+    }
+
+    public List<CarDto> getStatus() {
+        return cars.stream()
+                .map(CarDto::toDto)
+                .toList();
+    }
+
+    public List<String> getWinnerCarNames() {
+        final int farthestLocation = getFarthestPosition();
+        return cars.stream()
+                .filter(car -> car.isPositionSameWith(farthestLocation))
+                .map(Car::getName)
+                .toList();
+    }
+
+    private int getFarthestPosition() {
+        return Collections.max(this.getCurrentPositions());
+    }
+
+    private List<Integer> getCurrentPositions() {
+        return cars.stream()
+                .map(Car::getPosition)
+                .toList();
+    }
```

Comment:
위의 두 메서드는 하나의 스트림 체인으로도 구현하기 좋을 것 같습니다. 
현재 자동차들의 위치를 받아오는 기능이 다른 곳에서 필요하지도 않으니까요!
```suggestion
    private int getFarthestPosition() {
        return cars.stream()
                .mapToInt(Car::getPosition)
                .max()
                .getAsInt();
    }
```

Reply:
좋은 방법이네요! 공유해 주셔서 감사합니다. 

메서드당 하나의 기능을 계속 생각하며 구현하다 보니 조금 과하게 분리한 듯하네요. 😅